rslib/src/storage/card/get_costs_for_retention.sql

```diff
@@ -0,0 +1,44 @@
+WITH searched_revlogs AS (
+  SELECT *
+  FROM revlog
```

Comment:
```suggestion
  FROM revlog
  WHERE ease > 0
```

This would ensure that we use 10k user-graded revlog entries for calculating the costs.

Reply:
> does this use all the revlog entries in the collection or only those of cards in the preset

https://github.com/ankitects/anki/pull/3995/commits/2bd9c3461f602202e65c579707ccaf5416461343#diff-5d2f9fadb535a77bdb3e0db4b7a5341767eb1a0d09b8ce2bc17e0772f4ac3a10L4

I deleted this line by accident :neutral_face: